packages/tools/src/init.ts

```diff
@@ -74,6 +75,8 @@ function _addCornerstoneEventListeners(): void {
 
   eventTarget.addEventListener(elementEnabledEvent, addEnabledElement);
   eventTarget.addEventListener(elementDisabledEvent, removeEnabledElement);
+  console.log('Enabling cornerstone event listeners');
+  annotationInterpolationEventDispatcher.enable();
```

Comment:
Putting the dispatcher here means that it gets registered/unregistered once per lifetime of the CS tools, rather than ending up with multiple copies of the listener.